src/index.js

```diff
@@ -8,7 +8,6 @@ const createNewTaskElement = function todoItems(taskString) {
   const label = document.createElement('label');
   const editInput = document.createElement('input');
   const editButton = document.createElement('button');
```

Comment:
- Good job so far 👍 The add book functionality works well but there are some issues with the other functionalities which you need to address.
- [ ] The index of the tasks should begin from 1 and not 0. Kindly correct that 👍 
- [ ] After editing a task, it does not save on the local storage, and as a result after refreshing the page, the edited tasks go back to their previous description. This can be seen in the screenshot below. Take note how the description of the second task is different from that in the local storage.
- Your site:
![Screenshot 2022-07-09 195105](https://user-images.githubusercontent.com/85113567/178119232-aa03a809-e7ee-4bc1-a692-f242185a2872.png)

Reply:
- [ ] [REQUIRED] You have done a great job implementing most of the recommended changes. However, after editing a task, it still does not get updated in the local storage just as the previous reviewer pointed out. Kindly take a look at the screenshot I attached below and see how I have edited the `second` task to `new` but it does not get updated in the local storage. Please ensure that the local storage gets updated after a task has been edited just like you did after a task gets deleted in order to get this milestone approved as it is a core requirement

![Screenshot from 2022-07-09 22-55-25](https://user-images.githubusercontent.com/58692590/178123882-2331183a-42d5-4f3f-a6a3-4fd7a4e21e89.png)